content/docs/payment-providers/reepay/package.yml

```diff
@@ -0,0 +1,17 @@
+id: Vendr.PaymentProviders.Reepay
+name: Reepay
+logo: reepay.png
```

Comment:
Missing image is breaking the preview build. Image needs adding to`src\assets\images\logos`